packages/big-design/src/components/ButtonGroup/styled.tsx

```diff
@@ -0,0 +1,31 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled, { css } from 'styled-components';
+
+import { ButtonProps } from '../Button';
+import { StyleableButton } from '../Button/private';
+
+export const StyledButton = styled(StyleableButton)<ButtonProps & { isFirst?: boolean; isLast?: boolean }>`
+  border-radius: 0;
```

Comment:
Can use `theme.borderRadius.none` here ?

Reply:
Thank you for your comment 🙇 

Sure I can 😄 done ✅